frontend/src/app/containers/ExperimentPage/index.js

```diff
@@ -99,15 +83,15 @@ export class ExperimentDetail extends React.Component {
     }
   }
 
-  componentWillMount() {
+  UNSAFE_componentWillMount() {
     this.checkCookies();
   }
 
   componentDidUpdate() {
     this.checkCookies();
   }
 
-  componentWillReceiveProps(nextProps: ExperimentDetailProps) {
+  UNSAFE_componentWillReceiveProps(nextProps: ExperimentDetailProps) {
```

Comment:
Ditto

Reply:
React is deprecating these calls, it was showing up in the linter